App/Containers/OnboardingCheck.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ActivityIndicator, View } from 'react-native'
+import { connect } from 'react-redux'
+import { NavigationScreenProps } from 'react-navigation'
+
+import { RootState } from '../Redux/Types'
+
+interface Props extends NavigationScreenProps<{}> {
```

Comment:
Example of defining components props type. In this case, we also adopt NavigationScreenProps because this screen can live inside of a navigator.